### DISCUSSION TITLE
[Question][Hw2] Debugging method assignment2

### DISCUSSION THREAD

**Discussion by Jaegoomon** [prosyslab-classroom/cs348-information-security#500] [Q&A]
Hello, I have some questions about Assignment2.

When I implemented python file for DDoS attack. Then I can get those messages on my Attacker VM terminal.

![image](https://user-images.githubusercontent.com/40551890/114267739-62b7f080-9a38-11eb-9992-7225ee780a88.png)

I wonder that the messages above are right response and how can I check them? Actually I can't find any result in mongo db server.

I think it means there is something wrong. But I have no idea how can I check whether victim has requests from amp server or not?

**Comment by Altterisk** [comment]
You can run either tcpdump or Wireshark to catch packets at the VICTIM's end (I would very much recommend Wireshark over tcpdump though).
And just to be sure AMP need to receive at least 10 packets in 60 seconds in order to add the result to DB server, so your script should send at least that many packets. Took me a few hours wondering why there was nothing in my MongoDB before realizing that.

**Comment by Jaegoomon** [comment]
@Altterisk 
Oh thank you.

I tried to use wire shark on Victim VM to catch the packet, but I have no idea about how to do it.

Would you please give me some advices or references?...

**Comment by Altterisk** [comment]
You need to install Ubuntu-Desktop before using Wireshark since it's a GUI tool and can't be worked directly from terminal (which will cost extra spaces).
After that just run Wireshark with Admin privileges and start capturing packets. You might also want to set the filter to "ip.addr == 192.168.20.1", not needed, but it would help filtering out unrelated packets.

**Comment by Jaegoomon** [comment]
Thank you!!
